vortex-array/src/array/varbinview/mod.rs

```diff
@@ -135,22 +157,17 @@ impl VarBinViewArray {
         })
     }
 
-    pub fn plain_size(&self) -> usize {
-        (0..self.views.len() / VIEW_SIZE).fold(0usize, |acc, i| {
-            let view = self.view_at(i);
-            unsafe { acc + view.inlined.size as usize }
-        })
+    pub(self) fn view_slice(&self) -> &[BinaryView] {
```

Comment:
same, why `pub(self)`?

Reply:
I want to make this available only interrnally to this struct, didn't think it was necessary exposing this as an api